app/src/main/java/com/gh4a/activities/issues_label/IssueLabelListActivity.java

```diff
@@ -244,21 +237,18 @@ public boolean onPrepareActionMode(ActionMode mode, Menu menu) {
         public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
             switch (item.getItemId()) {
             case Menu.FIRST:
-                if (mLabel == mAddedLabel) {
-                    new AddIssueLabelTask(mLabel.editedName, mLabel.editedColor).schedule();
-                } else {
-                    new EditIssueLabelTask(mLabel.getName(), mLabel.editedName, mLabel.editedColor)
-                            .schedule();
-                }
+                if (mLabel == mAddedLabel) this.addIssue();
```

Comment:
Code style: Do not collapse if statements like this. Always use the full form with new lines and braces for both if and else branches. _Also remove `this.` prefix like mentioned earlier.